vault/resource_identity_group.go

```diff
@@ -128,11 +136,13 @@ func identityGroupUpdateFields(d *schema.ResourceData, data map[string]interface
 
 		// Member groups and entities can't be set for external groups
 		if d.Get("type").(string) == "internal" {
-			if externalMemberEntityIds, ok := d.GetOk("external_member_entity_ids"); !(ok && externalMemberEntityIds.(bool)) {
+			externalMemberEntityIds := d.Get("external_member_entity_ids")
+			if !externalMemberEntityIds.(bool) {
 				data["member_entity_ids"] = d.Get("member_entity_ids").(*schema.Set).List()
 			}
 
-			if externalMemberGroupIds, ok := d.GetOk("external_member_group_ids"); !(ok && externalMemberGroupIds.(bool)) {
+			externalMemberGroupIds := d.Get("external_member_group_ids")
```

Comment:
This wasn't required to fix the inconsistent plan error but we recommend using `d.Get` for booleans  :)

Reply:
Nice catch! Thanks for fixing